yesod-form/Yesod/Form/Functions.hs

```diff
@@ -62,7 +62,7 @@ import Yesod.Core
 import Yesod.Core.Handler (defaultCsrfParamName)
```

Comment:
Can you provide a patch version bump in the cabal file, and add a ChangeLog entry about the change?